packages/tasit-act/src/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./tasitAct";
```

Comment:
We should capitalize filenames like this one in `PascalCase` if they contain and export a class

Reply:
The same to `tasitAccount.js`?

Reply:
Yeah - thanks! And let's use `git mv` for this change